src/main/java/com/dreammaster/gthandler/GT_Loader_Wires.java

```diff
@@ -16,29 +16,30 @@ public void run() {
     private void registerWires() {
 
         // ID Range: 1200 - 4100
-        // Free IDs: 2030 - 4100
+        // Free IDs: 2050 - 4100 Do not trust this.
```

Comment:
probably better to delete this, point to the text in MetaTileEntityIDs, or point to the debug